packages/core/src/components/alert/alert.tsx

```diff
@@ -8,11 +8,21 @@ import clsx from 'clsx';
 })
 export class Alert {
   @Prop()
-  level: 'success' | 'info' | 'warning' | 'danger';
+  status: 'success' | 'info' | 'warning' | 'danger';
+
+  statusAlt = {
```

Comment:
Ja, een object zal zeker werken.. Maar we kunnen tegenwoordig beter: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map

En maak er meteen een private static property van, ipv een public instance property: `private static Map<string, string>`.

Reply:
en `alt` is ook niet helemaal eerlijk.. Het is geen alternatieve tekst.

Reply:
Ik dacht dat je dit niet zou willen omdat er ook andere talen dan Nederlands uit zouden moeten kunnen komen.

Reply:
Na een uitgebreide uitleg van Thomas over het gebruik van `new Map()` heb ik de boel omgebouwd naar een `statusMap` constructie, die ook nog eens `private static` is.